packages/ui/package.json

```diff
@@ -76,6 +77,8 @@
     "@typescript-eslint/parser": "7.4.0",
     "@vitejs/plugin-react": "4.2.1",
     "@vitest/coverage-v8": "1.4.0",
+    "clean-package": "2.2.0",
```

Comment:
remove this

Reply:
I've merged the `main` branch with this branch and `clean-package` isn't there anymore..

Reply:
`clean-package` is at the root level of the monorepo in `package.json`